src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentRequirement.kt

```diff
@@ -39,25 +39,12 @@ class IncidentRequirement(
   val agency: AgencyLocation,
 
   @ManyToOne(fetch = FetchType.LAZY)
-  @JoinColumn(name = "RECORD_STAFF_ID", updatable = false, nullable = false)
+  @JoinColumn(name = "RECORD_STAFF_ID")
   val recordingStaff: Staff,
 
-  @Column(name = "MODIFY_USER_ID", insertable = false, updatable = false)
-  @Generated
-  var lastModifiedUsername: String? = null,
-
-  @Column(name = "MODIFY_DATETIME", insertable = false, updatable = false)
-  @Generated
-  var lastModifiedDateTime: LocalDateTime? = null,
-
-  // ---- NOT MAPPED Columns ---- //
-  // All AUDIT data
-
+  @Column(name = "RECORD_DATE")
+  var recordedDate: LocalDateTime,
```

Comment:
note that we don't have a `@Generated` here as we want the value from DPS to go into our database